src/Frontend/Modules/ContentBlocks/Widgets/Detail.php

```diff
@@ -17,11 +19,17 @@ public function execute()
     {
         parent::execute();
 
-        $contentBlock = FrontendContentBlocksModel::get((int) $this->data['id']);
+        /** @var ContentBlockRepository $contentBlockRepository */
+        $contentBlockRepository = $this->get('doctrine.orm.entity_manager')->getRepository(ContentBlock::class);
+        $contentBlock = $contentBlockRepository->findOneByIdAndLocale(
```

Comment:
These two lines can now be replaced by `$this->get('content_blocks.repository.content_blocks')->findOneByIdAndLocale(`...